docs/kbs_attestation_protocol.md

```diff
@@ -234,43 +426,57 @@ The [JSON web token](https://jwt.io/) standard is adopted for the token. The for
 
     /* JWT payload custom claim set */
 
-    /* The HW-TEE's public key sent by KBC in the attestation evidence, which is valid within the validity period of the token. */
-    "tee-pubkey": {
-        "algo": "",
-        "pubkey-length": "",
-        "pubkey": ""
-    }
+    /*
+     * The HW-TEE's public key sent by KBC in the attestation evidence, which is
+     * valid within the validity period of the token.
+     * It follows the JWK format.
+     */
+    "tee-pubkey": $pubkey
 }.
 [Signature]
 ```
```

Comment:
Ditto.

Reply:
@sameo Just to be sure, the KBS is expected to send the token pre-encrypted, doesn't it? That is, three base64-URL strings (header, payload and signature) separated by dots. Is that right? If so, perhaps the docs should explicitly state that?

Reply:
That's correct @slp. I expanded the token section quite a bit to describe this. I also updated the openAPI description accordingly. Please have a look.